scipy/spatial/transform/rotation.py

```diff
@@ -872,3 +874,31 @@ def __getitem__(self, indexer):
                   array.
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def random(cls, num=None, state=None):
+        """Generate rotations uniformly distributed in 3D space.
```

Comment:
I don't think mentioning 3D space is correct here. I think "distributed uniformly" would be sufficient (intuitively it's clear what it means).